e2e/cypress/tests/integration/account_settings/profile/email_spec.ts

```diff
@@ -186,12 +202,12 @@ describe('Profile > Profile Settings > Email', () => {
 
         // # Close modal then logout
         cy.get('body').type('{esc}');
-        cy.uiOpenUserMenu('Log Out');
+        cy.uiOpenUserMenu({'Log Out': true});
 
         // # Wait for one second for the mail to be sent out.
         cy.wait(TIMEOUTS.ONE_SEC);
 
-        cy.getRecentEmail({username, email}).then((data) => {
+        cy.getRecentEmail({user: testUser}).then((data) => {
```

Comment:
The three calls to getRecentEmail should be `cy.getRecentEmail(testUser)` to match existing usage.